component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAuthenticator.java

```diff
@@ -153,6 +159,15 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
 
         String username = null;
         String tenantDomain = context.getTenantDomain();
+
+        Map<String, String> parameterMap = getAuthenticatorConfig().getParameterMap();
+        boolean showAuthFailureReason = Boolean.parseBoolean(parameterMap.get(
+                FrameworkConstants.SHOW_AUTHFAILURE_RESON_CONFIG));
+        boolean showAuthFailureReasonOnLoginPage = false;
+        if (showAuthFailureReason) {
+            showAuthFailureReasonOnLoginPage = Boolean.parseBoolean(
+                    parameterMap.get(FrameworkConstants.SHOW_AUTH_FAILURE_REASON_ON_LOGIN_PAGE_CONF));
+        }
```

Comment:
this code block can be moved to https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/18/files#diff-40859d27c102b1276ff19d9babc410c39e9b0df8eedbbbd1a75368d6b0ddca64R205 as we don't need to execute this logic if other errors such as `ERROR_NO_AUTHENTICATED_USER` occurs

Reply:
Addressed in 717cb21d3e50e206063e976737cd64c43a63f190